src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -352,12 +375,13 @@ def create_application(self, full_layout=True):
             'example_line': Buffer(is_multiline=True),
             'default_values': Buffer(),
             'symbols': Buffer(),
-            'progress': Buffer(is_multiline=False)
+            'progress': Buffer(is_multiline=False),
+            'scenarios': Buffer(is_multiline=True, read_only=True),
         }
 
         writing_buffer = Buffer(
             history=self.history,
-            auto_suggest=AutoSuggestFromHistory(),
+            auto_suggest=auto_suggest or AutoSuggestFromHistory(),
```

Comment:
Maybe we can change the default value of `auto_suggest=None` to `auto_suggest=AutoSuggestFromHistory()` in the method definition, which can make the logic clearer?